host/xtest/xtest_6000.c

```diff
@@ -1352,6 +1352,117 @@ static void xtest_tee_test_6014_single(ADBG_Case_t *c, uint32_t storage_id)
 
 DEFINE_TEST_MULTIPLE_STORAGE_IDS(xtest_tee_test_6014)
 
+#ifndef __ANDROID__
```

Comment:
Yes I think it should work, the only reason why I added this guard is because I didn't know how to hack `Android.mk` to build twice but with a different name and compiler flags ;) And, I have no Android environment available immediately.